docs/Ways-of-Working/Guides/Engineering/Staff-Engineering/index.md

```diff
@@ -0,0 +1,63 @@
+
+Individual project work should be managed just like any other Support Hub planned work. This will improve visibility of the work and provide opportunities for other Support Hub members to collaborate on readying or implementation, though it doesn't necessarily mean that other Support Hub members are available or expected to participate!
+
+### Research and investigations
+
+This could be research or investigation into how to solve a specific problem, which would typically be done with the intention of contributing to subsequent technical design sessions with another delivery team. This might also be done to support a Product Manager where a problem has not yet been assigned to a specific delivery team.
+
+As above, investigations should be managed and readied like any other Support Hub planned work. In this case the increased visibility could well result in opportunities for other Support Hub members to learn about relevant topics or contribute ideas that might not otherwise get heard.
+
+### Technical design support
+
+A staff engineer can use their existing domain or technology expertise to help a delivery team create a technical design for some upcoming work. In most cases, participating in technical design will involve taking part in a design session; this may be anything from an hour to multiple days, depending on the scope and complexity of the work being designed.
+
+Even when they lack technical expertise in a relevant area, participating in technical design sessions (or being updated with the relevant outcomes) can be an effective way for staff engineers to develop and maintain their domain knowledge across areas of our technical estate they are less familiar with.
+
+Design sessions should appear as events in the staff engineer's calendar, which should serve to communicate their availability. However, extended design sessions should be communicated in advance with the rest of the Support Hub, to help mitigate the impact of their absence.
+
+### Delivery team support
+
+A staff engineer will often spend some time embedded within a delivery team. This will often be a natural follow-up to help implement something they have helped in the design phase of. It could also be providing practical support to a team working with a particular technology or part of the domain that the staff engineer has expertise with, or it could simply be a way of providing additional software engineering resource to a team that needs it.
+
+Whilst working with another team, the staff engineer will use whatever mechanisms the team they are joining already uses to manage their work and record their time.
+
+!!! note
+    Given that staff engineers can work across any team, delivery teams should be aware that the staff engineer joining them to provide support may not be very familiar with their ways of working. Estimation is likely to be a particular challenge, as different groups of people will develop different shared subjective scales.
```

Comment:
I think this is quite an important note and is a good reason for teams to document their ways of working. If Staff Engineers could join different delivery teams multiple times in a year then it's extra important that these ways of working are documented and kept up-to-date to ensure these transitions are always smooth.